### ISSUE TITLE
Error 403 the doom of SmartTube?

### ISSUE THREAD

**Issue by gate1975mlm** [yuliskov/SmartTube#3757]
22.65 was perfect for like a day lol

Are we going to have to deal with this 403 error weekly?

I feel really bad for the SmartTube developer for having to attempt to fix this issue every other day.

At some point he may decide to just give up and end SmartTube development and that would really suck but how can we really blame him.

YouTube is really screwing us! 😡

**Comment by daspri** [comment]
Maybe related to Youtube's A/B testing of the anti-bot token?   Sometimes it is tested for, sometimes not.

https://github.com/TeamNewPipe/NewPipeExtractor/pull/1197

**Comment by Patraputt** [comment]
Updated to 22.67, nothing works.

**Comment by Blabla1987** [comment]
> Updated to 22.67, nothing works.

It's working again for me.

**Comment by castillofrancodamian** [comment]
The "stable" version does not work either.

**Comment by rpgdev** [comment]
The initial fix worked for me (22.38?), though it had the format and resolution limitation, but the dot 68 update which I just installed it since it claimed fixed the issue for good still shows the 403 error. The video eventually did play after 15s or so but it's very annoying! I prefer the resolution limitation. The most annoying part of that was that smart tube didn't remember the resolution I set but I could do with that.

**Comment by rpgdev** [comment]
I have to amend my last comment, now I'm not getting the error, happened with the first video I played after updating but haven't seen it again. I wonder if the people still facing issues can simply clear data and see how it goes (make a backup first!).

@castillofrancodamian you are right I'm on dot 65 too, thought it was 68 my bad. I'm using what I guess everyone else is using which is smart tubes update wizard. The wizard doesn't say it's beta though.

**Comment by castillofrancodamian** [comment]
> I have to amend my last comment, now I'm not getting the error, happened with the first video I played after updating but haven't seen it again. I wonder if the people still facing issues can simply clear data and see how it goes (make a backup first!).
> 
> @castillofrancodamian you are right I'm on dot 65 too, thought it was 68 my bad. I'm using what I guess everyone else is using which is smart tubes update wizard. The wizard doesn't say it's beta though.

I don't know how others update. I update by downloading the apk file and installing with Easy Fire Tools.

**Comment by jackL999** [comment]
> > Updated to 22.67, nothing works.
> 
> It's working again for me.

Version 22.67 no longer exists. The latest version available as of now is 22.65 beta

**Comment by Patraputt** [comment]
It seems YouTube is smarter than the smarttube. 🧐

**Comment by jackL999** [comment]
> It seems YouTube is smarter than the smarttube. 🧐

Not smarter, just literally a million times more resourceful given they are a multi-hundred-billion dollar corporation vs just one man.

**Comment by rpgdev** [comment]
> > I have to amend my last comment, now I'm not getting the error, happened with the first video I played after updating but haven't seen it again. I wonder if the people still facing issues can simply clear data and see how it goes (make a backup first!).
> > @castillofrancodamian you are right I'm on dot 65 too, thought it was 68 my bad. I'm using what I guess everyone else is using which is smart tubes update wizard. The wizard doesn't say it's beta though.
> 
> I don't know how others update. I update by downloading the apk file and installing with Easy Fire Tools.

Why? Once you have it installed you can update from smarttube itself. Unless it's a version shared out of that channel but looks like that's not being the case for these "beta" versions (not sure if this was communicated somewhere?).

**Comment by Patraputt** [comment]
> > It seems YouTube is smarter than the smarttube. 🧐
> 
> Not smarter, just literally a million times more resourceful given they are a multi-hundred-billion dollar corporation vs just one man.

You make me more concerned. 🥺

**Comment by isuckatthishelp** [comment]
Some users on telegram are claiming if you just wait a few seconds after the 403 smarttube fixes itself, at least that's what I read before I got banned before I could even say anything lol. Either way that's not working for me, this 403 does seem to be a lingering problem and I feel for the dev, but thank you for your dedication thus far.

**Comment by castillofrancodamian** [comment]
> Why? Once you have it installed you can update from smarttube itself. Unless it's a version shared out of that channel but looks like that's not being the case for these "beta" versions (not sure if this was communicated somewhere?).

The problem was that in the past you couldn't install apps from unknown sources. I had that option blocked by the decoder firmware. I was updating the app in the same way and now that I try updating from the same app it can now be updated on the same device. The installed version now is 22.67. But the playback error still appears.

**Comment by rpgdev** [comment]
> Some users on telegram are claiming if you just wait a few seconds after the 403 smarttube fixes itself

That's what happened to me. Haven't seen the issue again. But I'm also seeing that some videos are not displaying all the resolutions, showing avc resolutions only.. not sure why. Others show all the resolutions.

**Comment by isuckatthishelp** [comment]
> That's what happened to me. Haven't seen the issue again. But I'm also seeing that some videos are not displaying all the resolutions, showing avc resolutions only.. not sure why. Others show all the resolutions.

Okay, I just switched back to smarttube and it's suddenly working, can confirm I only have avc resolutions showing.

**Comment by castillofrancodamian** [comment]
> > That's what happened to me. Haven't seen the issue again. But I'm also seeing that some videos are not displaying all the resolutions, showing avc resolutions only.. not sure why. Others show all the resolutions.
> 
> Okay, I just switched back to smarttube and it's suddenly working, can confirm I only have avc resolutions showing.

I waited a few seconds when opening a video and I get the same results.

**Comment by Xeddius** [comment]
I'm also getting the 403 error for every video. Sick of Youtube and their greed, my kids don't need to be bombarded with ads when we're trying to listen to music. I will forever fight against their tyranny.

**Comment by NotMI5MI6** [comment]
Smart tube beta just undated in app again today 22.69 all good working fine on Firestick Next Gen

**Comment by diegoX-X** [comment]
It really pains me to say this but SmartTube has become unusable, at least for me. I'm running it on an older Android box and the only reason I'm not using firefox with 700 addons and extensions to make current youtube somewhat palatable is that it chugs like crazy. Both use cases are extremely frustrating because the user experience has been degraded to the point of having no other alternative. If by any means, any internet collective that commands some force and is willing to take on youtube **aggressively** is recuiting, I'm enlisting my dumb ass because I'm frankly fed up. **AND THIS IS NOT AN INDICTMENT/COMPLAIN DIRECTED TOWARDS THE DEV** I greatly appreciate his efforts because I would have said f youtube long ago.

**Comment by rpgdev** [comment]
> I don't like the 69 pose although I update to **.69** and ghost playback returned back. 🙄

What's ghost playback?

**Comment by rpgdev** [comment]
> If by any means, any internet collective that commands some force and is willing to take on youtube **aggressively** is recuiting, I'm enlisting my dumb ass because I'm frankly fed up. **AND THIS IS NOT AN INDICTMENT/COMPLAIN DIRECTED TOWARDS THE DEV** I greatly appreciate his efforts because I would have said f youtube long ago.

Stop derailing the conversation with your bs. If you meant this you'd support this project with either code or money. This is the 'collective' we have right now.

**Comment by Patraputt** [comment]
> > I don't like the 69 pose although I update to **.69** and ghost playback returned back. 🙄
> 
> What's ghost playback?


https://github.com/user-attachments/assets/c3be1d00-5ca3-4171-927b-7d22e9cdd897

Ghost playback 👻

**Comment by Patraputt** [comment]
It seems 22.70 currently fixes the issues.

**Comment by BobSmith43** [comment]
Does anyone found solution?

**Comment by Patraputt** [comment]
> Does anyone found solution?

Yup... **.70**

**Comment by wilberfan** [comment]
We need an **ERROR 403** t-shirt.  😏

**Comment by NotMI5MI6** [comment]
> We need an **ERROR 403** t-shirt. 😏

Good idea for a T-shirt definitely 😆 well at the end of the day shows us we shouldn't take the Guys who do the code for Newpipe and Smarttube for granted 👍 would you want to just be tied to youtube boxed in with the basic stuff 😬 My setup on next Gen Firestick

1. VLC Nightly 3.6.0 brilliant everyone try it all platforms streaming from ftp works also 
2. FILE MANAGER PLUS Alphainventor version brilliant for firestick easyer to navigate , setup ftp server or connect to phone pc whatever etc
3. Newpipe 0.27.2 all working since updating

With those apps all streaming file management even APK .text files .html app management is a simpler

**Comment by Percival9** [comment]
Error 403 also pops up for me but it only seems to pop up if I signed in with my youtube account.

**Comment by adz7** [comment]
STN v26.11 (issues started immediately after this update).
Nvidia Shield
Clear Cache, Clear Data (Lost all of my settings)..
Rebooted, no difference.  Only thing I haven't tried is complete uninstall/reinstall.

Get 403 Error on every video at 59seconds - wait for it to "Apply the fix" but doesn't work, just keeps trying to fix and looping.
